### ISSUE TITLE
LaTeX environments align and equation don't work

### ISSUE THREAD

**Issue by lluni** [silverbulletmd/silverbullet-katex#2]
The LaTeX environments align and equation don't work, example code:
```latex
\begin{equation}
    a = b
\end{equation}
```
This should render as a centered equation with the index `(1)` on the right side.

I haven't yet tested other environments.

**Comment by zefhemel** [comment]
Yeah, this is not a full latex rendering environment (unless KaTeX supports this, I haven’t looked too deeply into it). You should be able to put equations in triple-backtick `latex` blocks though:

    ```latex
     a = b
    ```
